spec/cocoapods/generate/podfile_generator_spec.rb

```diff
@@ -187,7 +187,7 @@
                    share_schemes_for_development_pods: true,
                    warn_for_multiple_pod_sources: false
 
-          use_frameworks!
+          use_frameworks!(false)
```

Comment:
this is now correct because the tests specify to use a podfile. The `Podfile` does not specify `use_frameworks!` in the tests.

Reply:
This is a breaking change, `pod gen` has defaulted to using frameworks